src/commctl/cli.py

```diff
@@ -277,6 +277,32 @@ def host_delete(self, address, **kwargs):
         path = '/api/v0/host/{0}'.format(address)
         return self._delete(path)
 
+    def cluster_deploy_status(self, name, **kwargs):
+        """
+        Attempts to get the status of an ongoing tree image deployment.
+
+        :param name: The name of the cluster
+        :type name: str
+        :param kwargs: Any other keyword arguments
+        :type kwargs: dict
+        """
+        path = '/api/v0/cluster/{0}/deploy'.format(name)
```

Comment:
Totally not something that needs to be done here but it hit me when reading this that we should probably have something like:

```python
_API_PREFIX = '/api/{version}'
_DEFAULT_VERSION = 0

def make_endpoint(version=_DEFAULT_VERSION, *path):
    endpoint = _API_PREFIX.format(version)
    for item in path:
        endpoint += '/{0}.format(item)
    return endpoint
```